community-operators/postgresql/4.1.0/postgresoperator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,549 @@
+    createdAt: 10/29/2019
+    description: A Postgres Operator from Crunchydata.com
+    repository: https://github.com/CrunchyData/postgres-operator
+    support: crunchydata.com
+  name: postgresoperator.v4.1.0
+  namespace: pgo
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Represents a Postgres primary cluster member
+      displayName: Postgres Primary Cluster Member
+      kind: Pgcluster
+      name: pgclusters.crunchydata.com
+      resources:
+      - kind: Deployment
+        name: ''
+        version: v1
+      - kind: ReplicaSet
+        name: ''
+        version: v1
+      - kind: Pod
+        name: ''
+        version: v1
+      - kind: Service
+        name: ''
+        version: v1
+      specDescriptors:
+      - description: The desired number of member Pods for the deployment.
+        displayName: Size
+        path: size
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podCount
+      statusDescriptors:
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase:reason
+      version: v1
+    - description: Represents a Postgres replica cluster member
+      displayName: Postgres Replica Cluster Member
+      kind: Pgreplica
+      name: pgreplicas.crunchydata.com
+      resources:
+      - kind: Deployment
+        name: ''
+        version: v1
+      - kind: ReplicaSet
+        name: ''
+        version: v1
+      - kind: Pod
+        name: ''
+        version: v1
+      - kind: Service
+        name: ''
+        version: v1
+      specDescriptors:
+      - description: The desired number of member Pods for the deployment.
+        displayName: Size
+        path: size
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podCount
+      statusDescriptors:
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase:reason
+      version: v1
+    - description: Represents a Postgres sql policy
+      displayName: Postgres SQL Policy
+      kind: Pgpolicy
+      name: pgpolicies.crunchydata.com
+      resources:
+      - kind: Pgpolicy
+        name: ''
+        version: v1
+      specDescriptors:
+      - description: The pgpolicy name.
+        displayName: Name
+        path: name
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:name
```

Comment:
If this path ("name") is expecting a `type: string`, using `urn:alm:descriptor:com.tectonic.ui:text`  (see [details in "Text Input"](https://github.com/openshift/console/blob/c07050bccf2e2c1d94001d138a063f6d4206043b/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md#text-input)) as the `x-descriptors` would expose this field as a "text input box" on the "Instance Creation Form" view in OKD console, and expose the info on "Operand's Details" view.